pack/fhv/fhvc.json

```diff
@@ -146,15 +146,296 @@
+    "back_text": "The lead investigator must decide (choose one):\n- Each undefeated investigator suffers 1 mental trauma.\n- Choose a resident under an investigator's control. That resident steps between you and the monstrosity, sacrificing themselves. In the Campaign Log, cross out the chosen resident's name. In that Resident's notes, record <i>[name] sacrificed themselves for the investigators</i>.\nAdvance the act.",
+    "code": "10628",
+    "doom": 10,
+    "double_sided": true,
+    "encounter_code": "the_longest_night",
+    "encounter_position": 3,
+    "faction_code": "mythos",
+    "flavor": "Your only hope is to last until dawn.",
+    "illustrator": "Mauro Dal Bo",
+    "name": "The Onslaught",
+    "pack_code": "fhvc",
+    "position": 128,
+    "quantity": 1,
+    "stage": 1,
+    "text": "When checking the doom threshold, doom on cards other than this agenda subtract from the total doom in play instead of adding to it.\n<b>Forced</b> – When a card effect would place doom on the agenda: Place it on The Captives story asset instead.\n<b>Forced</b> – After doom is placed on this agenda: In player order, each investigator draws the top card of the Enemy deck.",
+    "type_code": "agenda"
+  },
+  {
+    "back_name": "By the Skin of Your Skin",
+    "back_text": "<b>If 4 or fewer damage is on The Captives:</b>\n<blockquote><i>Despite the seemingly endless onslaught, relatively few of the captives were harmed. As you survey the wrecked interior of the farmhouse, some captives weep into their hands, and others look at you accusingly. How can you convince them that their lives were worth saving?</i></blockquote>\n<b>(→R1)</b> <i>(page 50)</i>.\n<hr>\n<b>Otherwise:</b>\n<blockquote><i>The farm is a charnel house of gore and viscera. When confronted with the onslaught, many of the captives ran, screaming, into the jaws of the monstrosities. The morning light illuminates a scene that will haunt your dreams.</i></blockquote>\n<b>(→R2)</b> <i>(page 50)</i>.",
+    "clues": null,
+    "code": "10629",
+    "double_sided": true,
+    "encounter_code": "the_longest_night",
+    "encounter_position": 4,
+    "faction_code": "mythos",
+    "flavor": "You must protect the captives!",
+    "illustrator": "Mauro Dal Bo",
+    "name": "The Longest Night",
+    "pack_code": "fhvc",
+    "position": 129,
+    "quantity": 1,
+    "stage": 1,
+    "text": "[free] During your turn, spend 1 clue: Place 1 barrier, decoy, or trap at your location.\n[action]: <b>Resign.</b> Then, if each undefeated investigator has resigned, <b>(→R3)</b> <i>(page 50).</i>\n<b>Objective</b> – Survive the night and protect as many captives as you can. <i>(Do not advance until you are instructed.)</i>",
+    "type_code": "act"
+  },
+  {
+    "back_flavor": "The aspect of the whole farm was shocking— greyish withered grass and leaves on the ground, vines falling in brittle wreckage from archaic walls and gables.\n<cite>H.P. Lovecraft, \"The Colour Out of Space\"</cite>",
```

Comment:
```suggestion
    "back_flavor": "The aspect of the whole farm was shocking - greyish withered grass and leaves on the ground, vines falling in brittle wreckage from archaic walls and gables.\n - H.P. Lovecraft, \"The Colour Out of Space\"",
```

Reply:
NB we don't use em dashes on the db, just a simple hyphen is fine.

Reply:
That's not exactly true, we are using em dashes a bunch in this very file and i.e. eote and tsk.

Reply:
> That's not exactly true, we are using em dashes a bunch in this very file and i.e. eote and tsk.

ok, please stick to dashes and I'll remove the em dashes on those two packs.